src/ruby/travelling-salesman.rb

```diff
@@ -12,11 +12,10 @@ def initialize(nodes, startMatrix)
     ##
     # shortest path function
     # need to fix matrix override
-    public
```

Comment:
Any method in a class is going to be public until the code hits the `private` or `protected` keywords. So having public in here isn't necessary. Conversely, say the code hits the keyword `private` a few methods down from here, it will assume any methods it encounters after this are supposed to be private, so putting private before each method isn't required. You could then add `public` above the methods you want to be public later on in your file if you so wished.